### ISSUE TITLE
Concept of "meta fields" for Bard & Replicator sets

### ISSUE THREAD

**Issue by duncanmcclean** [statamic/ideas#1182]
This came up a few times in my conversations with folks at Flat Camp.

On Bard & Replicator sets, it might be nice if it was possible to add extra meta/settings fields for sets, that aren't shown with the rest of the set's fields.

## Use case

You might have fields like "Background", "Animation style" etc, which aren't strictly content-related and you don't want to clutter the publish interface with fields that are rarely updated.

Instead of these fields being visible all the time, maybe we could add an option to the set's dropdown for the meta/settings fields which would toggle some kind of other section w/ other custom fields.

This feature probably needs to be thought out a little more but wanted to leave it here so it doesn't get forgotten about.

**Comment by jasonvarga** [comment]
I could see this somehow combined with @jacksleight 's field actions feature.

**Comment by andjsch** [comment]
True. Though it would be nice to be able to configure these additional settings within the blueprint builder. But hiding those settings within the field actions would be perfect.

**Comment by jasonvarga** [comment]
I agree, but I just mean the UI part of it. (Spoiler, Jack is already working on something)

![image](https://github.com/statamic/ideas/assets/105211/24789c06-4c06-417b-b372-406a6bf4ec82)
